pkg/integration/tests/stash/stash_staged_partial_file.go

```diff
@@ -0,0 +1,66 @@
+package stash
+
+import (
+	"github.com/jesseduffield/lazygit/pkg/config"
+	. "github.com/jesseduffield/lazygit/pkg/integration/components"
+)
+
+var StashStagedPartialFile = NewIntegrationTest(NewIntegrationTestArgs{
+	Description:  "Stash staged (all files and content) changes",
```

Comment:
I'm not sure what the "(all files and content)" means. Should we say something like "Stash staged changes when a file is partially staged"?

Reply:
Yes, remnants from the previous PR